contracts/manager/BaseManager.sol

```diff
@@ -180,9 +317,141 @@ contract BaseManager {
      * @param _module           Module to remove
      */
     function removeModule(address _module) external onlyOperator {
+        require(!protectedModules[_module].isProtected, "Module protected");
```

Comment:
Let's add a note that any extensions associated with the module need to be removed in a separate transaction

Reply:
1392513